INFO:root:Program started
INFO:root:Robot set up!
INFO:root:Odometry was started, PID: 17570:
INFO:root:Initial location: 0.6, 1.8, -1.5707963267948966
DEBUG:root:voy a girar
DEBUG:root:he girado, voy recto
WARNING:root:Traceback (most recent call last):
  File "/home/pi/oscar/main.py", line 34, in main
    mv.slalom_right(oscar, 0.4)
  File "/home/pi/oscar/actions/moves.py", line 167, in slalom_right
    run(robot, [origin[0] - tile_size/2, origin[1]- 3*tile_size/2])
  File "/home/pi/oscar/actions/moves.py", line 40, in run
    print('est√°: {} quiere llegar a: {}, is_near: {}'.format(math.sqrt(pow(robot.x.value * math.cos(th), 2) + pow(robot.y.value*math.sin(th), 2)), math.sqrt(pow(objctv[0]*math.cos(th), 2) + pow(objctv[1]*math.sin(th), 2))), helpers.location.is_near(robot, objctv, threshold=0.02))
IndexError: tuple index out of range

WARNING:root:Abruptly stopped the robot!
INFO:root:Odometry was stopped
DEBUG:matplotlib.axes._base:update_title_pos
DEBUG:matplotlib.font_manager:findfont: Matching :family=sans-serif:style=normal:variant=normal:weight=normal:stretch=normal:size=10.0 to DejaVu Sans ('/home/pi/.local/lib/python3.5/site-packages/matplotlib/mpl-data/fonts/ttf/DejaVuSans.ttf') with score of 0.050000.
DEBUG:matplotlib.axes._base:update_title_pos
